contributing/bazel_kokoro.md

```diff
@@ -117,22 +117,19 @@ If your component depends on non-source resources, you can add a bundle target t
 with the following additions:
 
 ```
+load("//tools/build_defs/apple:resources.bzl", "apple_bundle_import")
+
 mdc_public_objc_library(
     name = "ComponentName",
-    bundles = [":Bundle"],
```

Comment:
These changes appear unrelated, perhaps they should be split out into a separate PR

Reply:
https://github.com/material-components/material-components-ios/pull/6171